BEASTIE/__init__.py

```diff
@@ -8,12 +8,12 @@
 import logging
 import os.path
 from collections import namedtuple
+from pathlib import Path
 from datetime import date
-
 from . import beastie_step1, beastie_step2
 
 ConfigurationData = namedtuple("ConfigurationData", [
-    "prefix", "vcf_file_name", "vcf_sample_name", "pileup_file_name", "ancestry", "min_total_cov", "min_single_cov", "sigma", "cutoff", "alpha", "chr_start", "chr_end", "read_length", "LD_token", "modelName", "STAN", "work_dir", "ref_dir", "input_dir", "SAVE_INT", "WARMUP", "KEEPER", "output_dir"
+    "prefix", "vcf_file_name", "vcf_sample_name", "pileup_file_name", "ancestry", "min_total_cov", "min_single_cov", "sigma", "cutoff", "alpha", "chr_start", "chr_end", "read_length", "LD_token", "modelName", "STAN", "work_dir", "ref_dir", "input_dir", "SAVE_INT", "warmup", "keeper", "output_dir","output_folder"
```

Comment:
Just to keep them consistent with other parts of the code

Reply:
I guess that's my point -- in some places you have `WARMUP` and `KEEPER` and now in other places you have `warmup` and `keeper`. I think it would be best to pick one or the other.